schemas/src/main/resources/profileschema/0.1/datahelix.schema.json

```diff
@@ -350,7 +350,14 @@
                   1E-17,
                   1E-18,
                   1E-19,
-                  1E-20
+                  1E-20,
+                  "millis",
```

Comment:
Are these the correct case; are these required to be lower case?

Reply:
Yes the supported datetime units must be provided in lowercase:
Value is not accepted.
Valid values: 1, 0.1, 0.01, 0.001, 0.0001, 0.00001, 0.000001, 1e-7, 1e-8, 1e-9, 1e-10, 1e-11, 1e-12, 1e-13, 1e-14, 1e-15, 1e-16, 1e-17, 1e-18, 1e-19, 1e-20, "millis", "seconds", "minutes", "hours", "days", "months", "years".

We followed the ofType convention here:
Valid values: "decimal", "integer", "string", "datetime".